servicebus.tf

```diff
@@ -38,4 +38,15 @@ resource "azurerm_servicebus_subscription_rule" "allowed_jurisdictions" {
   subscription_name   = local.subscription_name
   filter_type         = "SqlFilter"
   sql_filter          = "1=1"
-}
\ No newline at end of file
+}
+
+resource "azurerm_servicebus_subscription_rule" "message_context" {
+  count               = var.env == "aat" ? 1 : 0
+  name                = local.subscription_rule_name
+  resource_group_name = local.resource_group_name
+  namespace_name      = local.servicebus_namespace_name
+  topic_name          = local.topic_name
+  subscription_name   = local.subscription_name
+  filter_type         = "SqlFilter"
+  sql_filter          = "message_context LIKE 'wa-ft%'"
```

Comment:
So I guess we only want to apply this if the env is `aat` as per line 44.  Will it only apply this if the count is 1?

Reply:
just want to make sure it doesn't get to prod

Reply:
To test this first I have changed the filter expression to always evaluate to true `1=1` to avoid any disruption  if this is successful we can add the expression in another PR.